bundles/binding/org.openhab.binding.zwave/database/products.xml

```diff
@@ -1,2600 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It seems that in this latest version, every line of the products file is marked as changed. Please can you change this back so we don't loose the history?